src/app/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import { phantomApi } from '../services/phantomApi';
```

Comment:
We can have two stores in one application. 
Please if you decide to go with `redux toolkit` stick to it.